sprocs/grading_jobs_insert_external_manual.sql

```diff
@@ -1,7 +1,10 @@
+DROP FUNCTION IF EXISTS grading_jobs_insert_external_manual(bigint, bigint);
+
 CREATE OR REPLACE FUNCTION
```

Comment:
Splitting the job is a +1 for me; I just wanted to establish the column split without trodding over _too_ much. If you split the sprocs in your branch, then it may be worth reverting this file, and just calling that sproc instead of passing the grading method type to this sproc (as one currently does).